documentation/book/kafka-exporter/con-kafka-exporter-lag.adoc

```diff
@@ -1,12 +1,12 @@
-// This assembly is included in the following assemblies:
+// Module included in the following assemblies:
 //
 // assembly-kafka-exporter.adoc
-[id='con-metrics-kafka-exporter-lag{context}']
+[id='con-metrics-kafka-exporter-lag-{context}']
 
 = Consumer lag
 
 Consumer lag indicates the difference in the rate of production and consumption of messages.
-Specifically, consumer lag indicates the delay between a Kafka broker writing a message created by a producer and the rate in which it is picked up by a consumer.
+Specifically, consumer lag indicates the delay between a Kafka broker writing a message created by a producer and the rate at which it is consumed up by a consumer.
```

Comment:
A delay between an _event_ (write message) and a _rate_ makes no sense. I think the definition is something like this.

```suggestion
Specifically, the lag for a given message and consumer is the delay between a Kafka broker writing that message to its log and the time at which a consumer's offset is incremented past that message's offset.
```